sdks/python/apache_beam/pipeline.py

```diff
@@ -28,15 +28,18 @@
 Typical usage:
 
   # Create a pipeline object using a local runner for execution.
-  pipeline = Pipeline(runner=DirectPipelineRunner())
+  p = beam.Pipeline('DirectPipelineRunner')
 
   # Add to the pipeline a "Create" transform. When executed this
   # transform will produce a PCollection object with the specified values.
-  pcoll = pipeline.create('label', [1, 2, 3])
+  c = p | 'label' >> beam.Create([1, 2, 3])
+
+  # Write to a file. The filename will be formatted as output-00000-of-00001.
```

Comment:
Do we need the write part for this little example? It does not add much to how pipeline is used beyond the previous example and needs to deal with file output formats.